src/routes/categoryRoutes.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
+
+const express = require('express');
+const {
+  createCategoryService,
+  getCategoryByIdService,
+  updateCategoryService,
+  deleteCategoryService,
+} = require('../services/categoryService');
```

Comment:
I think it would be ideal to export the whole service with methods on it, just to access them as follows:
`categoryService.createCategory`